openff/evaluator/protocols/reweighting.py

```diff
@@ -491,7 +491,9 @@ def _reweight_observables(
             observable_dimensions = observable.value.shape[1]
             assert observable_dimensions == 1
 
-            results = mbar.computeExpectations(
+            from openff.evaluator.utils.pymbar import compute_expectations
+
+            results = getattr(mbar, compute_expectations)(
                 observable.value.T.magnitude,
                 target_reduced_potentials.value.T.magnitude,
                 state_dependent=True,
```

Comment:
These lines aren't actually run in unit tests and the tests would need to be updated to feed data that converges if not bootstrapping uncertainties